### ISSUE TITLE
Integration with Azure javascript API 

### ISSUE THREAD

**Issue by chenfli** [microsoft/azure-devops-node-api#391]
Hi, 

_This is not an issue but a question_

My name is Chen and we are starting a migration process from Github to Azure DevOps.

I check this library and it's great, I am trying to find a complete solution to work with Azure Container Registry, Azure Resources and all of Azure stack.

I expect to be able to create a new project via Azure rest APIs
This project will contain several repositories, test plans, pipelines, artifacts. To create a full CI/CD solution it will create ACR with relevant registries as well as Helm charts to the git repositories.

I am a bit confused since this place holds all the Javascript APIs for azure DevOps but the rest of the Azure offering is not included. On the other hand, [Azure javascript SDK](https://azure.github.io/azure-sdk-for-js/index.html) exposed all the rest of Azure services as well as Browser and Server authentication.

Using Azure DevOps I can see there is a separation between Azure DevOps and the rest of Azure services. 

Is there a parallel SDK on the [Azure Github Account](https://github.com/azure)?

If not what is the recommended Library to use in order to have both Browser and Server authentication for all of the Azure offerings?

Thanks in advance

Chen

**Comment by stephenmichaelf** [comment]
Hey Chen,

I am not sure I understand the question. You use this library for Azure DevOps and the Azure SDK when working with Azure services. They are two separate libraries.

Could you elaborate on your question?

**Comment by chenfli** [comment]
With pleasure @stephenmichaelf,
I am trying to set up a migration plan from Github/Dockerhub to Azure DevOps.
Since we have a lot of git repositories and docker repositories, I need a way to create a new Azure devops projects which will have an Azure Container Registry for (charts/ container images).

I looked around to see if there is an SDK which supports such options (create Organziation/Project/Repositroies/ and Container Registries). But I failed to find one suits that does it all.

I did find this sdk(api) as well as [Azure javascript SDK](https://azure.github.io/azure-sdk-for-js/index.html). 

I just thought that it would be easier to use Azure DevOps if it was part of the Azure Javascript SDK, are you planning to add your library into the Azure JS SDK ?

If it is not on the road map of this repo/microsoft I will try to use both libraries. I believe it will be easier if there was one covnetion and integartion will be easier for me. 

Thanks for you time in anyway.
